controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -116,14 +116,6 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return reconcile.Result{}, err
 	}
 
-	// Extract latest Manifests
-	err = deploy.DownloadManifests(instance.Spec.DevFlags.ManifestsUri)
```

Comment:
do we want to remove this function totally? 
- modelmesh has not done transition, if we remove it, we can only merge this PR after they  finish the transion or even till we announce odh-manifests repo is archived 
- need put some word in the DSCI API saying this is not in use any more.